config/locales/simple_form.es.yml

```diff
@@ -0,0 +1,27 @@
+---
+es:
+  simple_form:
+    labels:
+      defaults:
+        avatar: Avatar
+        confirm_new_password: Confirmar nueva contraseña
+        confirm_password: Confirmar contraseña
+        current_password: Contraseña actual
+        display_name: Mostrar nombre
+        email: Dirección de correo electrónico
+        header: Título
+        locale: Idioma
+        new_password: Nueva contraseña
+        note: Biografía
+        password: Contraseña
+        username: Nombre de usuario
+      notification_emails:
+        favourite: Enviar correo electrónico cuando alguien de a favorito en tu publicación
+        follow: Enviar correo electrónico cuando alguien te siga
+        mention: Enviar correo electrónico cuando alguien te mencione
+        reblog: Enviar correo electrónico cuando alguien vuelva a publicar tu publicación
```

Comment:
"Vuelva a publicar" -> "Rebloguee" or "Comparta"
Have to standardize the Reblog translation.